02_bubble/eastrd/main_test.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+)
+
+func TestBbl1(t *testing.T) {
+	expected := []int{1, 2, 3, 4, 5}
+	actual := bubble([]int{5, 4, 3, 2, 1})
+
+	if (expected == nil) != (actual == nil) || len(expected) != len(actual) {
```

Comment:
`len(nil) == 0` so you could shorten the condition to:
`if len(expected) != len(actual)`

Use a different error message to the one below for easier debugging.

Error messages should start with lower case (see [official recommendations](https://github.com/golang/go/wiki/CodeReviewComments#error-strings)).

Last, at this stage it might be easier to use testify's [assert.Equal](https://godoc.org/github.com/stretchr/testify/assert#Equal) for slice comparison.